api/rest/property_value.md

```diff
@@ -6,156 +6,492 @@ parent: REST
 
-<iframe width="100%" height="300" src="//jsfiddle.net/datacommonsorg/285dcqb4/12/embedded/" allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"></iframe>
+## What to expect in the response
 
-## Success Response
-
-### **Code**: `200 OK`
-
-**Response content example**
+Your response will always look like this:
 
 ```json
 {
     "payload": "<payload string>",
 }
 ```
 
-The equivalent JSON of "payload string" is as follows.
+Here `"<payload string>"` is replaced by JSON, whose structure changes depending on whether the response contains node references.
 
-**Example 1:** property values that are not nodes.
+**Structure 1:** Response for property values that are not node references.
 
 ```json
 {
-    "geoId/05": {
-        "out": [
-          {
-            "value": "Arkansas",
-            "provenanceId": "dc/sm3m2w3"
-          }
-        ]
-    },
-    "geoId/06": {
-        "out": [
+    "dcid": {
+        "direction": [
           {
-            "value": "California",
-            "provenanceId": "dc/sm3m2w3"
-          }
+            "value": "string",
+            "provenanceId": "string"
+          },
+          ...
         ]
     },
+    ...
 }
 ```
 
-**Example 2:** property values that are node references.
+**Structure 2:** Response for property values that are node references.
 
 ```json
 {
-    "geoId/06":
+    "dcid":
         {
-        "in": [
+        "direction": [
             {
-              "dcid": "ipedsId/486619",
-              "name": "Academy of Professional Cosmetology",
-              "provenanceId": "dc/89fk9x3",
-              "types": ["CollegeOrUniversity"]
+              "dcid": "string",
+              "name": "string",
+              "provenanceId": "string",
+              "types": "string[]"
             },
-            {
-              "dcid": "ipedsId/485731",
-              "name": "The Beauty School",
-              "provenanceId": "dc/89fk9x3",
-              "types": ["CollegeOrUniversity"]
-            }
+            ...
         ]
+    },
+    ...
+}
+```
+
+**NOTES:** 
+ - The `provenanceId` is the DCID of the provenance for the corresponding value.
+ - You can run `JSON.parse()` on the `payload` field to retrieve the data. For example, in JavaScript: `var data = JSON.parse(response['payload'])`.
+
+## Example requests and responses
+
+### Example 1: Retrieve the order to which the plant _Austrobaileya scandens_ belongs.
+
+#### GET Request
+
+```curl
+curl --request GET \
+  --url 'https://api.datacommons.org/node/property-values?dcids=dc%2Fbsmvthtq89217&property=order'
+```
+
+#### POST Request
+
+```curl
+curl --request POST \
+  --url https://api.datacommons.org/node/property-values \
+  --header 'content-type: application/json' \
+  --data '{
+	"dcids": [
+		"dc/bsmvthtq89217"
+	],
+	"property": "order"
+}'
+```
+
+#### Response
```

Comment:
We should be very clear that this is after decoding, and that there is two steps involved, otherwise users will be very confused.

Reply:
Added that language.